### ISSUE TITLE
Use logging instead of print

### ISSUE THREAD

**Issue by j08lue** [ibamacsr/sentinelsat#83]
I would like to use `sentinelsat` in a operational script where I need to log to files. I could redirect output from the CLI, but it would be much more convenient if I could use Python's `logging` system.

Could we replace all the `print` statements by logging?

Single challenge I am seeing: How does homura do the status IO and how can that be logged?

I would be ready to take on this task, if you support this.

**Comment by valgur** [comment]
That would be nice to have, indeed.
Homura writes to `homura.STREAM` which defaults to `sys.stdout`. If it's important for you to also capture progress output from homura, then you could do something like

```python
import homura
import io

class HomuraIO(TextIOBase):
    def write(self, s):
        log.info(s)

homura.STREAM = HomuraIO()
```

It's also possible to set `show_progress=False` when initializing homura to just mute it.

**Comment by j08lue** [comment]
Excellent, thanks @valgur. 

@willemarcel do you think you will accept #81? Then I would build on top of that to simplify merging.

**Comment by Fernerkundung** [comment]
+1 on logging. I'd favour a clean cut where only the CLI prints something to stdout and everything done via Python uses logging - maybe with an added parameter to turn logging on/off.

**Comment by Fernerkundung** [comment]
Can this be closed after #93 @j08lue?
